doc/features.mld

```diff
@@ -106,13 +106,13 @@ end
 module type B = A
 ]}
 
-For these simple module type declarations, where the right-hand side is just a
+For these simple module-type declarations, where the right-hand side is just a
```

Comment:
I don't think OCaml ever uses dash between module and types: https://ocaml.org/manual/5.2/modtypes.html

```suggestion
For these simple module type declarations, where the right-hand side is just a
```

However, I might not know an English rule that makes this dash correct!

Reply:
Normally there wouldn't be a `-` between module and type, but in instances like these, a `-` is put between two words like this acting as a single adjective; however, I think I see my confusion. 

It's not a type of module definition (which would be module-type), it's a `module type` definition. Correct? The definition of the expression `module type`. Is this what you mean here?

Reply:
Correct - a "module type" is a different category of thing than a "module". I'd be happy with monospace or just simply "module type definition" - I don't think hyphenating is right in this context.